widgets-samples/cc/samples-cc-react-app/src/App.tsx

```diff
@@ -330,8 +330,8 @@ function App() {
                       setLoginType(selectedType);
                     }}
                   >
-                    <Option key={1} value="token">Access Token</Option>
-                    <Option key={2} value="oauth">Login with Webex</Option>
+                    <Option data-testid='samples:login_option_token' key={1} value="token">Access Token</Option>
+                    <Option data-testid='samples:login_option_oauth' key={2} value="oauth">Login with Webex</Option>
```

Comment:
Should we really complicate this value by having a ":"? Can we just do "_" or "-" every where?